.github/workflows/flutter-macos.yaml

```
@@ -16,6 +16,7 @@ concurrency:
 
 jobs:
   asr:
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)